docs/content/en/docs-dev/contribution-guidelines/contributing-documentation.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Contribute to PipeCD Documentation"
+linkTitle: "Contribute to PipeCD Documentation"
+description: >
+  A guide for how to contribute to the PipeCD Documentation website (https://pipecd.dev).

```

Comment:
```suggestion
This page describes how to contribute to the PipeCD Documentation.
```

I think the description is better off this way. The current drscp looks cluttered on the website. 

<img width="562" height="77" alt="Image" src="https://github.com/user-attachments/assets/6f405fa3-60d4-471f-bdb3-975d88c07995" />

Reply:
fix this nit, and we are good to go, @shivansh-gohem